src/dotnet/commands/dotnet-test/TestCommandParser.cs

```diff
@@ -40,19 +40,19 @@ internal static class TestCommandParser
                   Create.Option(
                         "-a|--test-adapter-path",
                         LocalizableStrings.CmdTestAdapterPathDescription,
-                        Accept.ExactlyOneArgument()
+                        Accept.OneOrMoreArguments()
                               .With(name: LocalizableStrings.CmdTestAdapterPath)
-                              .ForwardAsSingle(o => $"-property:VSTestTestAdapterPath={o.Arguments.Single()}")),
+                              .ForwardAsSingle(o => $"/p:VSTestTestAdapterPath=\"{string.Join(";", o.Arguments)}\"")),
```

Comment:
Any reasons changing from `-property` to `/p`?

Reply:
The CLI source base actually standardized on `-property`, so let's keep it as it was.

Reply:
Addressed the comment